ct/cmd/install.go

```diff
@@ -61,12 +61,15 @@ func addInstallFlags(flags *flag.FlagSet) {
 	flags.String("helm-extra-args", "", heredoc.Doc(`
 		Additional arguments for Helm. Must be passed as a single quoted string
 		(e.g. "--timeout 500 --tiller-namespace tiller"`))
+	flags.Bool("upgrade", true, heredoc.Doc(`
```

Comment:
The default should be `false` in order to not break the existing behavior.

Reply:
I'm good with this, though I hope we can flip this to true in a release or two. It would be nice for end users to get all the benefits of `ct` out of the box without having to figure out which additional flags they need to enable.